Packages/Tracking/Physical Hands/Runtime/Scripts/PhysicalHandsButtonBase.cs

```diff
@@ -0,0 +1,344 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Leap.Unity.PhysicalHands
+{
+    [RequireComponent(typeof(Rigidbody))]
+    public class PhysicalHandsButtonBase : MonoBehaviour
+    {
+        
+        [SerializeField, Tooltip("The GameObject representing the button that can be pressed.")] 
+        protected GameObject _pressableObject;
+        [SerializeField, Tooltip("The distance the button can travel when pressed. " +
+            "\n If left a 0, the travel distance will be the y distance from the parent object.")] 
+        private float _buttonTravelDistance = 0f;
```

Comment:
I wonder if this could be some sort of "automatic" distance, with an ability to override it - rather than a "leave it as 0" tooltip